src/threadpool.c

```diff
@@ -310,3 +332,44 @@ int uv_cancel(uv_req_t* req) {
 
   return uv__work_cancel(loop, req, wreq);
 }
+
+
+static void report(enum estage stage, int lock) {
+  QUEUE* q;
+  uv_queue_stats_t* s;
+  int length_;
```

Comment:
Could you name these just `length` and `threads`.

Reply:
I was trying to avoid shadowing the file-scope var called `threads`, but gcc doesn't seem to care, so I renamed.